roles/ovirt-host-deploy-libvirt-guests/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Fetch info about VDSM
+  command: bash -c "rpm -qi vdsm | grep -oE 'Version\\s+:\\s+[0-9\\.]+' | awk '{print $3}'"
```

Comment:
1. See https://github.com/oVirt/ovirt-ansible/pull/56/files
2. I guess we should extract that VDSM version task to its own file, so it can be reused?

Reply:
This is probably better written as 
command: 'rpm -q --qf "%{version}" vdsm'

Reply:
Ondro is there a way how to fetch VDSM version in ovirt-host-deploy role itself and afterwards store it in some variable and pass it to all subroles?

@machacekondra

Reply:
As long as it's in the same playbook (you can also do this as a role dependency) then variables set in one role will be available in another.

Reply:
Right. We can create something like hosts-facts role, which will be executed first.

Reply:
I have created the issue for this task so we do not forget about it. #70 
Does this patch need to wait for the #70 ? If not, can we move forward with the merge?
@machacekondra @mwperina @mykaul

Reply:
If you plan to move forward I would strongly urge you to replace the command with 

`  command: "rpm -q --qf '%{version}' vdsm"`

it's massively more readable than the grep+awk combination

Reply:
There is PR adding that new role:

 https://github.com/oVirt/ovirt-ansible/pull/71

Feel free to review.